src/components/Lend/AaveAction.tsx

```diff
@@ -0,0 +1,46 @@
+import { TxBuilderV2, Network, Market } from "@aave/protocol-js";
+import { ethers } from "ethers";
+
+export const deposit = async (provider, accountAddress) => {
+  let txBuilder;
+  let lendingPool;
+
+  let customProvider = new ethers.providers.Web3Provider(provider);
```

Comment:
Nit: I think all these local variables can all be `const`

Reply:
updated